src/main/kotlin/ar/edu/unahur/obj2/caralibro/Publicacion.kt

```diff
@@ -3,14 +3,88 @@ package ar.edu.unahur.obj2.caralibro
 import kotlin.math.ceil
 
 abstract class Publicacion {
-  abstract fun espacioQueOcupa(): Int
+    abstract fun espacioQueOcupa(): Int
+
+    val meGustaRecibidos = mutableListOf<Usuario>()
+    fun cantidadMeGusta(): Int { return meGustaRecibidos.size }
```

Comment:
Si utilizan la sintaxis recomendada por la materia, no hace falta explicitar el tipo que devuelve:

```suggestion
    fun cantidadMeGusta() = meGustaRecibidos.size
```